src/writing-policies/go/02-scaffold.md

```diff
@@ -0,0 +1,74 @@
+# Creating a new validation policy
+
+We are going to create a validation policy that validates the labels of generic Kubernetes
+objects.
+
+The policy will reject all the resources that use one or more labels on the deny list.
+The policy will also validate certain labels using a regular expression 
+provided by the user.
+
+To summarize, the policy settings will look like that:
+
+```yaml
+# List of labels that cannot be used
+denied_labels:
+- foo
+- bar
+
+# Labels that are validate with user-defined RegExp
+# Failing to comply with the RegExp resuls in the object
+# being rejected
```

Comment:
This comment does not seem to hold together. Different sentences?

Reply:
solved, thanks